src/routes/subscribe/add.ts

```diff
@@ -0,0 +1,60 @@
+import { RequestHandler } from 'express';
+
+import Subscriber from '../../models/Subscriber';
+import validateEmail from '../../utils/validateEmail';
+
+const addSubscriber: RequestHandler = async (req, res) => {
+    try {
+        const { eAddress } = req.body;
+        const email = eAddress.toLowerCase();
```

Comment:
change this so `req.body` would have a field called `email` 
and rename email to eAddress